changes/8da40a8252615ed2243a907c06ab165a.yaml

```diff
@@ -0,0 +1,6 @@
+---
+desc: Prevent the ``synapse.tools.promote`` from promoting a service into a leader
```

Comment:
Maybe we could tighten this up a little? Maybe something like:

Update ``synapse.tools.promote`` to prevent graceful promotion when the mirror is not following the current leader.

Or... something... :)